### PR TITLE
TL-Layer: Update To Layer 71

### PR DESCRIPTION
**Pull request by aarani**
This Layer needed lots of Manual Working to Update so don't confuse if generator didn't work same

### CODE REVIEW

File: TeleSharp.TL/TL/Channels/TLRequestGetAdminLog.cs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    {
+        public override int Constructor
+        {
+            get
+            {
+                return 870184064;
+            }
+        }
+
+        public int flags { get; set; }
```
Comment (knocte) [inline TeleSharp.TL/TL/Channels/TLRequestGetAdminLog.cs]: @aarani do you mind fixing this to be PascalCase? It's not respecting the MS .NET API guidelines.

File: TeleSharp.TL/TL/Channels/TLRequestGetAdminLog.cs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        public TLAbsInputChannel channel { get; set; }
+        public string q { get; set; }
+        public TLChannelAdminLogEventsFilter events_filter { get; set; }
```
Comment (knocte) [inline TeleSharp.TL/TL/Channels/TLRequestGetAdminLog.cs]: Another glitch: members should be PascalCase with no underscores

**Comment by aarani** [comment]
Ok, So we have to update generator first. We'll check that

File: TeleSharp.TL/TL/Channels/TLRequestGetAdminLog.cs (hunk 1 above)
Comment (aarani) [reply TeleSharp.TL/TL/Channels/TLRequestGetAdminLog.cs]: Pascal or Camel Case ?

Comment (knocte) [reply TeleSharp.TL/TL/Channels/TLRequestGetAdminLog.cs]: camelCase is for Java (lowercase the first letter, uppercase each new word), but for .NET it is PascalCase (uppercase the first letter of each word, even the first one)

Comment (knocte) [reply TeleSharp.TL/TL/Channels/TLRequestGetAdminLog.cs]: so it should be "Flags" instead of "flags", and "EventsFilter" instead of "events_filter".

Comment (knocte) [reply TeleSharp.TL/TL/Channels/TLRequestGetAdminLog.cs]: (for method names, properties and public fields; private fields can be left as camelCase)

Comment (aarani) [reply TeleSharp.TL/TL/Channels/TLRequestGetAdminLog.cs]: #627

**Pull request closed without merging**